src/api/app/views/webui/package/_buildstatus.html.erb

```diff
@@ -1,4 +1,4 @@
-<% @package.buildresults.each_pair do |package, results| %>
+<% @package.buildresults(@project).each_pair do |package, results| %>
```

Comment:
Can we do this in a way that doesn't add more logic to the view? i.e. do this in the controller?

```ruby
@buildresults = @package.buildresults(@project)
```